services/web/project/__init__.py

```diff
@@ -0,0 +1,22 @@
+from flask import Flask, jsonify
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Missing docstring in public package](https://app.codacy.com/manual/mpgabriel95/TCC/pullRequest?prid=5161591)